tests/unit/environments/BinaryLabelData.h

```diff
@@ -0,0 +1,86 @@
+#ifndef BINARY_LABEL_DATA_HPP
+#define BINARY_LABEL_DATA_HPP
+
+#ifdef HAVE_LAPACK
```

Comment:
mmm oh that's on the end of the file (sorry haven't seen it)... yeah we should change this :( although the problem is not only DataGenerator but as well `Gaussian` distribution implementation as that one is required to have LAPACK as well.

actually this should be a more pressing issue, i.e. convert Gaussian to use modern linalg backend instead of LAPACK. are you up for that @MikeLing ?